mitiq/executor/tests/test_executor.py

```diff
@@ -199,53 +208,27 @@ def test_run_executor_preserves_order(s, b):
 )
 def test_executor_evaluate_float(execute):
     q = cirq.LineQubit(0)
-    circuits = [cirq.Circuit(cirq.X(q)), cirq.Circuit(cirq.H(q), cirq.Z(q))]
+    circuits = [
+        cirq.Circuit(cirq.X(q), cirq.M(q)),
+        cirq.Circuit(cirq.H(q), cirq.Z(q), cirq.M(q)),
+    ]
 
     executor = Executor(execute)
 
     results = executor.evaluate(circuits)
-    assert np.allclose(results, [1, 2])
+    assert np.allclose(results, [2, 3])
```

Comment:
Do we need to have this line, and the line below checking `quantum_results`?

Reply:
No. Deleting this line.

Reply:
Resolved with #1333ebf